frontend/public/style/_common.scss

```diff
@@ -1,12 +1,3 @@
-.co-alert {
-  max-height: 75px;
-  overflow-x: hidden;
-  overflow-y: auto;
-  @media (min-height: $screen-sm-min) {
-    max-height: 150px;
-  }
```

Comment:
I think we want to keep the ability for vertical scroll on alerts, because we ran into an issue where long alert messages would cause the yaml editor to be pushed off the screen and unusable.  Do you agree @spadgett ?

 https://jira.coreos.com/browse/CONSOLE-1348
https://jira.coreos.com/secure/attachment/19050/Screen%20Shot%202019-03-21%20at%201.11.18%20PM.png

Reply:
+1 we shouldn't just remove that. Note that before it was opt-in instead of applied to every alert

Reply:
Renamed to co-scrollable-alert and re-added to the ones that used to have it.

Reply:
@sg00dwin Would this be instead of allowing an alert to expand and collapse or did we previously support the scroll as well because the expanded version can get too long?

Reply:
@beanh66 This would still allow the alert to expand/collapse but give the ability to set a `max-height` and enable scroll.

Reply:
Gotcha! I would be okay without the scrollbar as long as we have the expand option so users are choosing to open the lengthy error message and can use the built in page scroll. They would still be able to collapse the error again right?

Reply:
Oh, I think I misread your previous comment. The alerts associated with the YAML editor did NOT previously have the toggle link option to expand/collapse.  But that would be an alternative to using this scrollable class,  setting a `max-height` and enable scroll inside the alert message.  ...  I see that ExpandableAlert component recently merged, but it looks only be enabled when there are multiple related alerts.